code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -53,4 +53,21 @@
 		else
 			user << "<span class='danger'>Access Denied.</span>"
 	else
-		return ..()
\ No newline at end of file
+		return ..()
+
+/obj/structure/closet/secure_closet/personal/verb/verb_resetlock()//personal locker ownership reset.
+	set src in oview(1)
+	set category = "Object"
+	set name = "Reset Lock"
+
+	if(opened)
+		if(broken)
+			usr << "<span class='danger'>The lock appears to be broken.</span>"
+			return
+		else
+			registered_name = null
+			usr << "<span class='danger'>You successfully reset the lock.</span>"
+			desc = initial(desc)
+			add_fingerprint(usr)
+	else
+		usr << "<span class='danger'>The [src.name] must be open!</span>"
```

Comment:
Sorry about the long return in response, had some RL things that needed to be taken care of.